examples/console/main.go

```diff
@@ -0,0 +1,218 @@
+	}
+	signerKey, err := keypair.ParseFull(signerKeyStr)
+	if err != nil {
+		return fmt.Errorf("cannot parse -signer: %w", err)
+	}
+
+	horizonClient := &horizonclient.Client{HorizonURL: horizonURL}
+	networkDetails, err := horizonClient.Root()
+	if err != nil {
+		return err
+	}
+
+	account, err := horizonClient.AccountDetail(horizonclient.AccountRequest{AccountID: accountKey.Address()})
+	if horizonclient.IsNotFoundError(err) {
+		fmt.Fprintf(os.Stdout, "account %s does not exist, attempting to create using network root key\n", accountKey.Address())
+		err = createAccountWithRoot(horizonClient, networkDetails.NetworkPassphrase, accountKey)
+		if err != nil {
+			return err
+		}
+		account, err = horizonClient.AccountDetail(horizonclient.AccountRequest{AccountID: accountKey.Address()})
+	}
+	if err != nil {
+		return err
+	}
+	accountSeqNum, err := account.GetSequenceNumber()
+	if err != nil {
+		return err
+	}
+
+	escrowAccountKey := keypair.MustRandom()
+	fmt.Fprintln(os.Stdout, "escrow account:", escrowAccountKey.Address())
+
+	submitter := &Submitter{
+		HorizonClient:     horizonClient,
+		NetworkPassphrase: networkDetails.NetworkPassphrase,
+		BaseFee:           txnbuild.MinBaseFee,
+		FeeAccount:        accountKey,
+		FeeAccountSigners: []*keypair.Full{signerKey},
+	}
+	agent := &Agent{
+		NetworkPassphrase:   networkDetails.NetworkPassphrase,
+		HorizonClient:       horizonClient,
+		Submitter:           submitter,
+		EscrowAccountKey:    escrowAccountKey.FromAddress(),
+		EscrowAccountSigner: signerKey,
+		LogWriter:           os.Stderr,
+	}
+
+	tx, err := txbuild.CreateEscrow(txbuild.CreateEscrowParams{
+		Creator:        accountKey.FromAddress(),
+		Escrow:         escrowAccountKey.FromAddress(),
+		SequenceNumber: accountSeqNum + 1,
+		Asset:          txnbuild.NativeAsset{},
+	})
+	if err != nil {
+		return fmt.Errorf("creating escrow account tx: %w", err)
+	}
+	tx, err = tx.Sign(networkDetails.NetworkPassphrase, signerKey, escrowAccountKey)
+	if err != nil {
+		return fmt.Errorf("signing tx to create escrow account: %w", err)
+	}
+	err = submitter.SubmitFeeBumpTx(tx)
+	if err != nil {
+		return fmt.Errorf("submitting tx to create escrow account: %w", err)
+	}
+
+	br := bufio.NewReader(os.Stdin)
+	for {
+		fmt.Fprintf(os.Stdout, "> ")
+		line, err := br.ReadString('\n')
+		if err == io.EOF {
+			fmt.Fprintf(os.Stdout, "connection terminated\n")
+			break
+		}
+		if err != nil {
+			fmt.Fprintf(os.Stdout, "error: %#v\n", err)
+			continue
+		}
+		params := strings.Fields(line)
+		if len(params) == 0 {
+			continue
+		}
+		switch params[0] {
+		case "help":
+			fmt.Fprintf(os.Stdout, "listen [addr]:<port> - listen for a peer to connect\n")
+			fmt.Fprintf(os.Stdout, "connect <addr>:<port> - connect to a peer\n")
+			fmt.Fprintf(os.Stdout, "open - open a channel with asset\n")
+			fmt.Fprintf(os.Stdout, "deposit <amount> - deposit asset into escrow account\n")
+			fmt.Fprintf(os.Stdout, "pay <amount> - pay amount of asset to peer\n")
```

Comment:
since balances are all in stroops except for here

```suggestion
			fmt.Fprintf(os.Stdout, "deposit <lumen amount> - deposit asset into escrow account\n")
			fmt.Fprintf(os.Stdout, "pay <lumen amount> - pay amount of asset to peer\n")
```

Reply:
Good point. I've changed all the amounts to be consistent. d0d7b94